File: cpp/dentro_fora.cpp
#include <iostream>

using namespace std;

int main() {

    int N, X, fora, dentro;

    cout << "Quantos numeros voce vai digitar? ";
    cin >> N;

    fora = 0;
    dentro = 0; //intervalo [10,20]
    for(int i = 0; i < N; i++){
        cout << "Digite um numero: ";
        cin >> X;

        if(X < 10 || X > 20){
            fora++;
        }
        else{
            dentro++;
        }
    }

    cout << dentro << " DENTRO" << endl;
    cout << fora << " FORA" << endl;

    return 0;
}

File: python/dentro_fora.py
N: int
x: int
fora: int
dentro: int

N = int(input("Quantos numeros voce vai digitar? "))

fora = 0
dentro = 0
for i in range(N):
    x = int(input("Digite um numero: "))

    if x < 10 or x > 20:
        fora += 1
    else:
        dentro += 1

print(f"{dentro} DENTRO")
print(f"{fora} FORA")
